qa/rpc-tests/spentindex.py

```diff
@@ -81,7 +79,7 @@ def run_test(self):
         assert_equal(tx1['vin'][0]['value'], 10)  # coinbase
         assert_equal(tx1['vin'][0]['valueSat'], 10*COIN)
         # we want the non-change (payment) output
-        vout = filter(lambda o: o['value'] == 2, tx1['vout'])
+        vout = list(filter(lambda o: o['value'] == 2, tx1['vout']))
```

Comment:
Not a blocker, but I think list comprehensions generally make a more readable pattern:

`vout = [x for x in tx1['vout'] if x['value'] == 2]`